aerolyzer/image_restriction_conf.yaml

```diff
@@ -0,0 +1,48 @@
+# Image Restriction Configuration File
+# Description: This configuration file stores information regarding image criteria that will be evaluated in the image restrictions code.
+
+---
+
+acceptedFileTypes:
+ - JPEG
+ - JPG
+ - PNG
+
+acceptedMobileDevices:
+ - i5
+ - i5s
+ - i6
+ - i6s
+ - i7
+
+imgSizesLong:
+ 0: bytes
+ 1: kilobytes
+ 2: gigabytes
+ 3: megabytes
+ 4: terrabytes
+
+imgSizesShort:
+ 0: b
+ 1: kb
+ 2: gb
+ 3: mb
+ 4: tb
+
+imgMaxSizeNumber: 200
+imgMaxSizeBytesShort: kb
+imgMaxSizeBytesLong: kilobytes
+imgWidthMin: 100
+imgHeightMin: 100
+imgWidthMax: 1000
+imgHeightMax: 1000
+imgRestrictionErrorText:
+ is_accepted_device: "The image must be a mobile image from a supported device"
```

Comment:
These vars 'is_*' should not be in the config file. Instead in the code, you should use assertions to test. See [this tutorial](https://www.tutorialspoint.com/python/assertions_in_python.htm) for more direction.